cmd/up/activate.go

```diff
@@ -356,7 +356,18 @@ func (up *upContext) waitUntilDevelopmentContainerIsRunning(ctx context.Context,
 				}
 				continue
 			}
-
+			pod, ok := event.Object.(*apiv1.Pod)
+			if !ok {
```

Comment:
what is this loop for? I would just ignore the event if we cannot get the pod of the event

Reply:
That's for StatefulSet. We can get events from the previous pod and it will say that's its pulling images but it's still attaching pvs. This will check that the pod of the event is the one that we need